src/EdFi.AnalyticsMiddleTier.DataStandard31/Base/MSSQL/0035-View-StudentSchoolDim-Alter.sql

```diff
@@ -0,0 +1,132 @@
+            StudentSchoolAssociation.SchoolId = School.SchoolId
+    LEFT OUTER JOIN
+        edfi.StudentEducationOrganizationAssociation AS studentEdOrg ON
+            Student.StudentUSI = studentEdOrg.StudentUSI
+            AND StudentSchoolAssociation.SchoolId = studentEdOrg.EducationOrganizationId
+    LEFT OUTER JOIN
+        edfi.Descriptor AS LimitedEnglishDescriptorSchool ON
+            studentEdOrg.LimitedEnglishProficiencyDescriptorId = LimitedEnglishDescriptorSchool.DescriptorId
+    LEFT OUTER JOIN
+        edfi.Descriptor AS SexTypeSchool ON
+            studentEdOrg.SexDescriptorId = SexTypeSchool.DescriptorId
+    LEFT OUTER JOIN
+        edfi.StudentEducationOrganizationAssociation AS districtEdOrg ON
+            Student.StudentUSI = districtEdOrg.StudentUSI
+            AND School.LocalEducationAgencyId = districtEdOrg.EducationOrganizationId
+    LEFT OUTER JOIN
+        edfi.Descriptor AS LimitedEnglishDescriptorDist ON
+            districtEdOrg.LimitedEnglishProficiencyDescriptorId = LimitedEnglishDescriptorDist.DescriptorId
+    LEFT OUTER JOIN
+        edfi.Descriptor AS SexTypeDist ON
+            districtEdOrg.SexDescriptorId = SexTypeDist.DescriptorId
+    LEFT OUTER JOIN
+        edfi.StudentEducationOrganizationAssociationStudentIndicator AS InternetAccessInResidence ON
+            studentEdOrg.StudentUSI = InternetAccessInResidence.StudentUSI
+            AND
+            studentEdOrg.EducationOrganizationId = InternetAccessInResidence.EducationOrganizationId
+            AND
+            (InternetAccessInResidence.StudentUSI IS NULL or InternetAccessInResidence.IndicatorName = 'InternetAccessInResidence')
+    LEFT OUTER JOIN
+        edfi.StudentEducationOrganizationAssociationStudentIndicator AS InternetAccessTypeInResidence ON
+            studentEdOrg.StudentUSI = InternetAccessTypeInResidence.StudentUSI
+            AND
+            studentEdOrg.EducationOrganizationId = InternetAccessTypeInResidence.EducationOrganizationId
+            AND
+            (InternetAccessTypeInResidence.StudentUSI IS NULL or InternetAccessTypeInResidence.IndicatorName = 'InternetAccessTypeInResidence')
+    LEFT OUTER JOIN
+        edfi.StudentEducationOrganizationAssociationStudentIndicator AS InternetPerformance ON
+            studentEdOrg.StudentUSI = InternetPerformance.StudentUSI
+            AND
+            studentEdOrg.EducationOrganizationId = InternetPerformance.EducationOrganizationId
+            AND
+            (InternetPerformance.StudentUSI IS NULL or InternetPerformance.IndicatorName = 'InternetPerformance')
+    LEFT OUTER JOIN
+        edfi.StudentEducationOrganizationAssociationStudentIndicator AS DigitalDevice ON
+            studentEdOrg.StudentUSI = DigitalDevice.StudentUSI
+            AND
+            studentEdOrg.EducationOrganizationId = DigitalDevice.EducationOrganizationId
+            AND
+            (DigitalDevice.StudentUSI IS NULL or DigitalDevice.IndicatorName = 'DigitalDevice')
```

Comment:
```suggestion
            DigitalDevice.IndicatorName = 'DigitalDevice'
```

Reply:
I'll change these in bulk, there are 4 files.

Reply:
Updated